stdlib/3/subprocess.pyi

```diff
@@ -2,37 +2,42 @@
 
 # Based on http://docs.python.org/3.6/library/subprocess.html
 import sys
-from typing import Sequence, Any, AnyStr, Mapping, Callable, Tuple, IO, Optional, Union, List, Type
+from typing import Sequence, Any, AnyStr, Mapping, Callable, Tuple, IO, Optional, Union, List, Type, Text
 from types import TracebackType
 
+_FILE = Union[int, IO[Any]]
```

Comment:
You can actually pass file descriptors (i.e., ints) to the stdout and stderr arguments in both Python 2 and 3. Didn't know about this either, but it seems to work.